samples/subsys/llext/edk/ext1/toolchain.cmake

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CMAKE_C_COMPILER   arm-zephyr-eabi-gcc)
+set(CMAKE_FIND_ROOT_PATH $ENV{ZEPHYR_SDK_INSTALL_DIR}/arm-zephyr-eabi)
```

Comment:
You mean, include the compiler itself? If so, that seem dangerous - what if some application developer is using some proprietary toolchain? I agree that the toolchain needs to be the same for application and extension developers, but that's part of their "contract". 
Or are you talking about something else?

Reply:
Hmmm - but that would assume Zephyr SDK is in use (due `ZEPHYR_SDK_INSTALL_DIR` env var). This won't always be the case, I kinda feel this would be confusing - to ship the EDK with a flag that doesn't always make sense.